### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by intoro** [savoirfairelinux/santropol-feast#827]
### Expected Behaviour
When I run:
 docker-compose build

### Actual Behaviour

ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a version of "2" (or "2.0") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

**Comment by erozqba** [comment]
@intoro Sorry for this, we have been using "dev" branch and "master" was really behind. I have merged dev into master, so this should work now. Could you test again?

**Comment by kousu** [comment]
@intoro That error message usually comes up for me when *docker* is out of date. Sous-Chef has been following the latest docker, and you really need to [follow](https://docs.docker.com/install/linux/docker-ce/debian/) [the](https://docs.docker.com/install/linux/docker-ce/ubuntu) [install](https://docs.docker.com/install/linux/docker-ce/centos) [instructions](https://docs.docker.com/install/linux/docker-ce/fedora), including those for [docker-compose](https://docs.docker.com/compose/install/), which means adulterating your system with the Docker PPAs (or equivalent), in order to get the latest version.

Sorry about that.

**Comment by kousu** [comment]
Did you give up on trying it?

**Comment by intoro** [comment]
No,  Just Been Busy.  I think I'll have time tonight.  I'll Update You

**Comment by kousu** [comment]
Thanks. Take your time and don't get discouraged! We really appreciate the feedback. This software is deployed but it's definitely a beta thing, so we are hungry for data on how it goes for others.

**Comment by erozqba** [comment]
@intoro is this now fixed?
